examples/react-native/src/ui/components/authenticator/sign-up-with-attributes/Example.tsx

```diff
@@ -14,6 +14,41 @@ Amplify.configure({
   Auth: { endpoint: 'http://127.0.0.1:9091/' },
 });
 
+const formFields = {
+  signUp: {
+    zoneinfo: {
+      label: 'Zone Info',
+      isRequired: false,
+      type: 'text',
+    },
+    picture: {
+      label: 'Picture',
+      isRequired: false,
+      type: 'text',
+    },
+    updated_at: {
+      label: 'Updated at',
+      isRequired: false,
+      type: 'text',
+    },
+    address: {
+      label: 'Address',
+      isRequired: false,
+      type: 'text',
+    },
+    gender: {
+      label: 'Gender',
+      isRequired: false,
+      type: 'text',
+    },
+    locale: {
+      label: 'Locale',
+      isRequired: false,
+      type: 'text',
+    },
```

Comment:
Not sure how much value including these in the example code provides, imo kind of bloats the example code. Are the keys not documented?

Reply:
Is this example used in the docs?

Reply:
No, docs use this [example](https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/%5Bplatform%5D/connected-components/authenticator/configuration/signUpAttributes.all.react-native.mdx)

Reply:
Got it, thanks!